posts/key-learnings-from-waku-hackers-in-2023

```diff
@@ -0,0 +1,79 @@
+
+[Václav](https://twitter.com/vpavlin), [Aaron](https://twitter.com/vrycmfy), and [Guru](https://twitter.com/hackyguru) represented the Waku team on the Space. We were also joined by participants from the winning ETHRome teams [Feka](https://twitter.com/Feka_7heaven), [mrnmrc.eth](https://twitter.com/mrnmrceth), [Martin](https://twitter.com/martinlecl), [Régis](https://twitter.com/Rgis295478), [Florian](https://twitter.com/floriantruchot), and [Mattia](https://twitter.com/Mattia).  
+
+[embed: https://twitter.com/Waku_org/status/1722659158816612463]
+
+
+## “Blockchain seems like the default way to do decentralised things, but in the end, it's not the only one” Mattia
+
+Mattia expressed his enthusiasm for the emergence of decentralised technologies like Waku that break away from the conventional reliance on blockchain. By operating independently from a blockchain, Waku benefits from decentralisation and speed without requiring gas fee payments to execute transactions. 
+
+Mattia praised the seamless interoperability enabled by Waku’s blockchain-agnostic nature, allowing his team and all the hackers at the event to use the protocols on any platform without restrictions. He noted that this helps to eliminate silos between chains, which is a key issue in web3. We believe this platform-agnostic characteristic will help Waku become the standard for communication in web3. 
+
+
+## Top tip for hackers: What we mean when we say “no more chat apps”
+
+As you may have noticed, during Waku’s recent hackathon, we requested that participants not submit ‘standard chat apps’. If you are wondering why, many of these are already in the ecosystem, and we want to encourage hackers to find new use cases that can add value to users. Chat apps refer to person-to-person messaging apps, such as Telegram. But much of the functionality within Waku revolves around chat-like interactions. As Guru and Valcav noted, in essence, Waku activities are akin to a chat interface, albeit one that transcends traditional UI constraints. Even sophisticated applications, such as that submitted by Mattia's team, are essentially exchanges of messages between JS Waku nodes, where the content could range from simple text to more complex data like GPS locations.
+
+Whether it is a conversation between humans or machines, the core principle remains a form of chat. Within the Waku ecosystem, users essentially engage in a dialogue, either issuing commands to nodes for execution or communicating with AI services, which then carry out tasks based on predefined transactions. Hackers require creativity to transform these chat-like exchanges into something truly exciting. 
```

Comment:
I'd focus on the fact that Waku enables "interaction".

The other reason of "no chat app" is that it is very hard to build a chat protocol with the right security parameters and we do not believe this is something that can be done over a hackathon.

Reply:
Good to know- updated. It would be great to get another review on teh updated content in this section